Spring_Server/gs-uploading-files-complete/src/main/java/compile_io/FileUploadController.java

```diff
@@ -57,26 +57,8 @@ public FileUploadController(StorageService storageService) {
 
     	// Docker stuff
 		File fileToUpload = new File(workingDir);
-		String temp = runCompiler(fileToUpload, "java");
```

Comment:
Hey I need lines 60-79, Everything else looks good

Reply:
got it